src/data-types.ts

```diff
@@ -396,6 +396,18 @@ export interface FrontendExerciseNode {
   href?: string
 }
 
+export interface FrontendSolutionNode {
```

Comment:
There is now some duplication with FrontendExerciseNode, maybe add FrontendSolutionNode to FrontendExerciseNode?